src/test/java/com/codeus/winter/config/DefaultBeanFactoryTest.java

```diff
@@ -708,6 +708,25 @@ void testShouldThrowExceptionWhenRegisterBeanDefinitionWithSameName() {
 
     }
 
+    @Test
+    @DisplayName("Should initialize a bean with multiple candidates using @Qualifier annotation")
+    void testShouldInitializeBeanWithMultipleCandidatesUsingQualifier() {
+        BeanDefinition beanDefinitionWithQualifierAnnotation = singletonBeanDefinitionMock(
+                BeanWithQualifierAnnotation.class
+        );
+        beanDefinitionWithQualifierAnnotation.setBeanClassName("com.codeus.winter.test.BeanWithQualifierAnnotation");
+        HashMap<String, BeanDefinition> beanDefinitionHashMap = new HashMap<>();
+        beanDefinitionHashMap.put("BeanA", beanDefinitionA);
+        beanDefinitionHashMap.put("BeanE", beanDefinitionE);
+        beanDefinitionHashMap.put("BeanWithQualifier", beanDefinitionWithQualifierAnnotation);
+
+        DefaultBeanFactory factory = new DefaultBeanFactory(beanDefinitionHashMap);
+
+        BeanWithQualifierAnnotation beanWithQualifierAnnotation = factory.getBean(BeanWithQualifierAnnotation.class);
+        assertNotNull(beanWithQualifierAnnotation);
```

Comment:
Please remove an empty line on 722 and add an empty line after `BeanWithQualifierAnnotation beanWithQualifierAnnotation = ...`. 
By doing that we separate test case setup, main action and result assertion.

Reply:
Fixed